django/otto/models.py

```diff
@@ -314,18 +338,48 @@ def get_user_cost_today(self, user):
         )
 
 
+FEATURE_CHOICES = [
+    ("librarian", _("Librarian")),
+    ("qa", _("Q&A")),
+    ("chat", _("Chat")),
+    ("translate", _("Translate")),
+    ("summarize", _("Summarize")),
+    ("template_wizard", _("Template wizard")),
+    ("laws_query", _("Legislation search")),
+    ("laws_load", _("Legislation loading")),
+    ("case_prep", _("Case prep assistant")),
+    ("text_extractor", _("Text extractor")),
+]
```

Comment:
We can extend this later, but it will result in a (harmless, if only adding choices) migration.